tests/function-controller/cmd/main.go

```diff
@@ -1,23 +1,24 @@
 package main
 
 import (
+	"errors"
 	"math/rand"
 	"os"
+	"time"
 
-	"github.com/kyma-project/kyma/tests/function-controller/pkg/step"
-	"github.com/kyma-project/kyma/tests/function-controller/testsuite"
-	"github.com/kyma-project/kyma/tests/function-controller/testsuite/scenarios"
 	"github.com/sirupsen/logrus"
 	"github.com/vrischmann/envconfig"
 	"k8s.io/client-go/rest"
 
-	"time"
+	"github.com/kyma-project/kyma/tests/function-controller/pkg/step"
+	"github.com/kyma-project/kyma/tests/function-controller/testsuite"
+	"github.com/kyma-project/kyma/tests/function-controller/testsuite/scenarios"
 
 	controllerruntime "sigs.k8s.io/controller-runtime"
 )
 
 var availableScenarios = map[string][]testSuite{
-	"serverless-integration": {scenarios.SimpleFunctionTest},
+	"serverless-integration": {scenarios.SimpleFunctionTest, scenarios.GitopsSteps},
```

Comment:
`gitopssteps` does not work due to the fact that we create DestinationRule there